Merge/mainnet-readiness.md

```diff
@@ -127,6 +134,9 @@ Most research has been completed. Only listing things still left to dig deeper i
     * [ ] Sync during transition period
 * [ ] Discovery [is there actually anything to do here?]
 * [ ] Execution-layer proof of custody
+* [ ] Disaster recovery if invalid chain finalized 
+  * [x] [WIP] Client multiplexers ([link](https://github.com/karalabe/minority))
```

Comment:
note, this doesn't help with the disaster recovery wrt consensus/validators. if validators use it, it might avoid the disaster in the first place